lib/tandem/tandemSimulator.js

```diff
@@ -85,6 +90,14 @@ exports.make = function(config){
           events.push(currBasal);
         }
       }
+
+      if(event.deliveryType === 'suspend') {
+        setCurrStatus('suspended');
+      }else if(event.deliveryType === 'resume') {
```

Comment:
Can we try to stick to a consistent style on if/elses? The rest of the codebase has:
```
if (blah) {
}
else {
}
```
With separate blocks and space after the `if` and `else` before the parens.